#1 = find/go to bricks, 2 = find/go to place to stack
int64 goal
----
int64 result 
----
int64 dummy
